.github/workflows/integration-tests.yaml

```diff
@@ -1,73 +0,0 @@
-name: integration-tests
-on:
-  push:
-    paths-ignore:
-      - '**.md'
-  pull_request:
-    paths-ignore:
-      - '**.md'
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        java: [ '8', '11' ]
-        architecture: [ 'x64' ]
-    name: Integration tests for CLI
-    steps:
-      - uses: actions/checkout@v2.3.2
-      - name: Setup JDK
-        uses: actions/setup-java@v1
-        with:
-          java-version: ${{ matrix.java }}
-          architecture: ${{ matrix.architecture }}
-
-      - uses: actions/cache@v2
-        with:
-          path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
-          restore-keys: ${{ runner.os }}-maven-
-
-      - name: Make
-        run: cd distro && make
-
-      - name: Set HZ_VERSION
-        run: |
-          echo "HZ_VERSION=$(grep '<hazelcast.version>' pom.xml | sed "s@.*<hazelcast.version>\(.*\)</hazelcast.version>.*@\1@")" >> $GITHUB_ENV
-
-      - name: Run CLI for Hazelcast
-        run: distro/build/dist/hazelcast-${{ env.HZ_VERSION }}/bin/hz start -c src/test/resources/integration-test-hazelcast.yaml &
```

Comment:
I wonder if it makes sense to modify the workflow to install a package from a local file and then test if it works. Do we have anything like that?

Reply:
In the #110 after we upload the packages we install them and try to run Hz.

https://github.com/hazelcast/hazelcast-command-line/pull/110/files#diff-f649dfa0883378db13b96d378fbaedfaf99e75a92d0962d813575b79643dd01bR106

(`verify-deb`, `verify-rpm`, `verify-homebrew steps`).

I plan to enable this workflow to build on push to this repository and build with latest version of Hz so there will be a test which builds a package and tries to install for changes in this repository.

Reply:
Yes, just saw it, thanks!